string name
time t
duration deltat
float64[] position
float64[] velocity
float64[] torque_gc
float64[] torque_ct
float64[] torque_imp
float64[] torque_imp_null
float64[] torque_imp_null_fric
float64[] torque_external
float64[] torque
float64[] Kp_Rn
float64[] Kd_Rn
float64[] Kp_SE3
float64[] Kd_SE3
float64 Kp_null
float64 Kd_null
float64 friction_gain
float64 impulse
float64 forceExternal
std_msgs/Float64MultiArray pose
float64[] setpointRn
float64[] setpointRnvelocity
float64[] setpoint_null
std_msgs/Float64MultiArray setpointSE3
float64[] setpointSE3velocity
int32 controller
float64[] erroraxis
float64 errorangle
float64 m_y
std_msgs/Float64MultiArray M
std_msgs/Float64MultiArray M_OS
std_msgs/Float64MultiArray M_JS
float64 tracking_goal
